src/Test.UnitTests.Sarif/Core/LocationTests.cs

```diff
@@ -85,5 +92,68 @@ public void Location_LogicalLocation_WhenSetToNull_RemovesLogicalLocationsArray(
 
             location.LogicalLocations.Should().BeNull();
         }
+
+        [Fact]
+        public void Location_SerializeId()
```

Comment:
>Location_SerializeId [](http://example.com/codeflow?start=20&length=20)

These tests may not provoke round-tripping issues?

Reply:
change to round trip